tests/mmptest/src/NetStandardTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Xamarin.MMP.Tests
+{
+	[TestFixture]
+	public partial class NetStandardTests
+	{
```

Comment:
Super minor but it looks like you don't have the right C# formatting settings :P